README.md

```diff
@@ -95,13 +112,27 @@ In particular, since the metadata description resides in a separate file, it is
 
 ## CLDF Ontology
 
-CLDF data uses terms from the [CLDF Ontology](http://cldf.clld.org/v1.0/terms.rdf), as specified in the file `terms.rdf`, to mark [`TableGroup`](http://w3c.github.io/csvw/metadata/#table-groups) or [`Table`](http://w3c.github.io/csvw/metadata/#tables) objects which have special meaning within the CLDF framework.
-
-The CLDF Ontology also provides a set of [properties](http://cldf.clld.org/v1.0/terms.rdf#properties) to attach semantics to individual columns. While many of these properties are similar (or identical) to properties defined elsewhere - most notably in the [General Ontology for Linguistic Description - GOLD](http://linguistics-ontology.org/) - we opted for inclusion to avoid ambiguity, but made sure to reference the related related properties in the Ontology.
-
-Note that the column *names* in the default table descriptions (e.g. [`formTable`](components/forms)) are not always the same as the column *properties*. Each column has both a `csvw:name` and a separate `propertyURL` linking the column to the ontology. Each property also has a `rdf:label` which might also be different.
-
-Note: For better human readability the [CLDF Ontology](http://cldf.clld.org/v1.0/terms.rdf) should
+CLDF data uses terms from the [CLDF Ontology](http://cldf.clld.org/v1.0/terms.rdf), as specified in the file `terms.rdf`, 
```

Comment:
Is this intended to be a link to `v1.0` (and to be later updated to `v1.1`)?

Reply:
No. Elsewhere it says:

> In an ill-advised attempt to version the ontology, `v1.0` has been baked into the term
  URIs. While this may be a good idea in case of incompatible changes (e.g. if the semantics
  of a term changed), it presents an obstacle for initeroperability in case of backwards-compatible
  changes. So starting with CLDF 1.1, we will keep `http://cldf.clld.org/v1.0/terms.rdf`
  as namespace for **all** versions of the 1.x series, and specify the particular version
  when a term was introduced using `dc:hasVersion` properties per term.

Reply:
Ah, sorry, understood.